src/styles/TodoItem.module.css

```diff
@@ -0,0 +1,35 @@
+.item {
```

Comment:
I'd suggest you align InputTodo and TodoItem on the left as well.
![image](https://github.com/manzitresor/react_todo_app/assets/24216198/0692764f-9ae5-4958-91b1-2df788c23dbb)